src/main/java/ToDo.java

```diff
@@ -0,0 +1,11 @@
+public class ToDo extends Task {
+
+    public ToDo (String task) {
+        super(task);
+    }
+
+    @Override
+    public String toString() {
+        return "[T]" + super.toString();
```

Comment:
I would recommend to have the label 'T' as a static constant outside the method.